frontend/lib/generate-image-markup.ts

```diff
@@ -45,7 +45,7 @@ function quoteAsciiDocAttribute(attr: string | null): string {
   if (attr == null) {
     return 'None'
   } else {
-    const withQuotesEscaped = attr.replace(/"/g, '\\"')
+    const withQuotesEscaped = attr.replace(/"/g, '\\"') // lgtm [js/incomplete-sanitization]
```

Comment:
What’s the thinking here, that we don’t care about other things that might need to be escaped? Probably worth adding a comment explaining why the ignore is being placed.

Reply:
There are two aspects here:
* why the ignore is being placed: LGTM is flagging it up as a potential security vulnerability. Here the string values are prepared and returned to the user they want to use their badge in a AsciiDoc document. We're not using the value in any actual processing, so no security risk (e.g. SQL injection, cross site scripting) as far as I can tell.
* why the existing code only escapes quotes: there's a link to the AsciiDoc User Guide at the point of call of this method (line 69 of this same class) which explains that "To include a double quotation mark (") character in a quoted attribute value the the quotation mark must be escaped with a backslash. ".

Reply:
That makes sense to me. (Heh, I think I wrote this code!) Could you add that in a comment?

Reply:
Done! :)